src/views/Pools/Syrup.tsx

```diff
@@ -33,69 +41,54 @@ const Farm: React.FC = () => {
   )
 
   return (
-    <Page>
-      <Hero>
-        <div>
-          <Heading as="h1" size="xxl" mb="16px">
-            {TranslateString(738, 'Syrup Pool')}
-          </Heading>
-          <ul>
-            <li>{TranslateString(580, 'Stake CAKE to earn new tokens.')}</li>
-            <li>{TranslateString(486, 'You can unstake at any time.')}</li>
-            <li>{TranslateString(406, 'Rewards are calculated per block.')}</li>
-          </ul>
-        </div>
-        <img src="/images/syrup.png" alt="SYRUP POOL icon" width={410} height={191} />
-      </Hero>
-      <PoolTabButtons stakedOnly={stakedOnly} setStakedOnly={setStakedOnly} />
-      <Divider />
-      <FlexLayout>
-        <Route exact path={`${path}`}>
-          <>
-            {stakedOnly
-              ? orderBy(stakedOnlyPools, ['sortOrder']).map((pool) => <PoolCard key={pool.sousId} pool={pool} />)
-              : orderBy(openPools, ['sortOrder']).map((pool) => <PoolCard key={pool.sousId} pool={pool} />)}
-            <Coming />
-          </>
-        </Route>
-        <Route path={`${path}/history`}>
-          {orderBy(finishedPools, ['sortOrder']).map((pool) => (
-            <PoolCard key={pool.sousId} pool={pool} />
-          ))}
-        </Route>
-      </FlexLayout>
-    </Page>
+    <>
+      <PageHeader>
+        <Flex justifyContent="space-between">
+          <Flex flexDirection="column" mr={['8px', 0]}>
+            <Heading as="h1" size="xxl" color="secondary" mb="24px">
+              {TranslateString(999, 'Syrup Pools')}
+            </Heading>
+            <Heading size="md" color="text">
+              {TranslateString(999, 'Simply stake tokens to earn.')}
+            </Heading>
+            <Heading size="md" color="text">
+              {TranslateString(999, 'High APR, low risk.')}
+            </Heading>
+          </Flex>
+          <Button
+            px={['14px', null, null, null, '24px']}
+            variant="subtle"
+            as="a"
```

Comment:
Use `Link` instead, and apply the `external` props

Reply:
Made that change here Doge https://github.com/pancakeswap/pancake-frontend/pull/887/commits/f77afef3b57c2bdb8e9f596e4d09d107b9a9fffa

Had to apply a couple of styles to the `Link` to prevent it filling the container